fs/xfstests.py.data/nvdimm_log.yaml

```diff
@@ -12,12 +13,13 @@ setup:
             #share_exclude: '1-300'
         fs_xfs:
             fs: 'xfs'
-            mkfs_opt: '-b size=65536 -s size=4096'
+            mkfs_opt: '-b size=65536 -s size=512 -m reflink=0'
```

Comment:
why this ymal changes i feel this is just a template user can have there owned YAML file

Reply:
These options are necessary for nvdimm type disk, which makes sure mount always works.